jetty-server/src/main/config/etc/jetty-secure-redirect.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Get id="Handler" name="handler"/>
+  <Call name="setHandler">
+    <Arg>
+      <New class="org.eclipse.jetty.server.handler.HandlerList">
+        <Call name="addHandler">
+          <Arg>
+            <New id="SecuredRedirectHandler" class="org.eclipse.jetty.server.handler.SecuredRedirectHandler">
+              <Arg type="int"><Property name="jetty.redirect.code" default="302"/></Arg>
```

Comment:
`301` (a permanent redirect) is probably the best default here.

I also wonder if the property should have a more specific name because if we are going to write another, different, redirect module, `jetty.redirect.code` could clash.
Perhaps `jetty.redirect.secure.status.code`?

Reply:
how about `jetty.secure.redirect.*` ?

Reply:
The default is 302 (Moved Temporarily) in code.
If we change it here in the XML it should be changed in `SecureRedirectHandler` too.

Reply:
From a Google perspective the response codes work like this ...

Permanent Redirect - show the new redirect target in search results.

* 301 (moved permanently)
* 308 (moved permanently)

Temporary Redirect - show the source page in search results

* 302 (found)
* 303 (see other) 
* 307 (temporary redirect)

Reply:
> how about jetty.secure.redirect.* ?

No, I prefer to be `jetty.<feature/module>.<other>`. `secure` is related to redirect (like status code), so it should be after `redirect`.

Reply:
The class is called `SecureRedirect` it is specific for going `http` to `https` only.
It could have been called "ensure secure" or `ConfidentialTransportGuaranteeHandler` as well.
The "redirect" in that is just a hint on _how_ it does it.  That word "redirect" should be nowhere near the properties or module name.

No different than a servlet constraint enforcing a confidential transport-guarantee. (this is the exact words in the servlet spec, it also has no mention of "redirect")

Reply:
So what should I change the property name to, or is `jetty.redirect.code` ok?

I could do something like `jetty.secureredirect.code`.

Reply:
Don't like "secured" with the "d" at the end. Can we drop it from the class name and the property name?
I like `jetty.secureredirect.*`.